include/ada/url_search_params-inl.h

```diff
@@ -18,6 +18,47 @@
 
 namespace ada {
 
+inline void url_search_params::initialize(std::string_view input) {
+  if (input.front() == '?') {
```

Comment:
Possibly check for empty view.

Reply:
I added a test for it, but it is not needed to check for it

Reply:
it seems like visual studio requires it :-)